python/GafferImageUI/CatalogueUI.py

```diff
@@ -437,6 +442,27 @@ def __incrementImageIndex( catalogue, direction ) :
 	if nextPlugIndex != currentPlugIndex :
 		indexPlug.setValue( nextPlugIndex )
 
+
+## \todo Work out how we want to restructure this code such that hot keys and
+# _ImageListing can re-use the same implementation.
```

Comment:
Are there any particular hurdles to doing this? If not, then let's do it.

Reply:
Amended [as discussed](https://github.com/GafferHQ/gaffer/pull/3871/commits/73dae627013deb31bbabc57bf2728dc74cdfcd0e).